debian/changelog

```diff
@@ -1,26 +1,67 @@
-gnome-control-center (1:3.32.1-1endless1) unstable; urgency=medium
+gnome-control-center (1:3.34.0.1-1endless1) unstable; urgency=medium
+
+  * Upgrade to upstream release 3.34.0.1
+  * Reapply "Remove .pkla file"
+  * Reapply "Use dh_eoscontent to merge Endless desktop customizations"
+  * Reapply "Add git to build-dep"
+  * Reapply "Add build dependency on eos-metrics-0-dev"
+  * Reapply "Add malcontent as dependency", squashing the change from
+    eos-parental-controls into malcontent in
+  * Reapply "Add udisks2 >= 2.1.8 dependency"
+  * Reapply "Add libmogwai-tariff-0 dependency"
+  * Reapply "control: Drop desktop-base dependency"
+  * Dropped the vendor config definition from debian/rules, as we no longer have
+    a vendor config file after this rebase.
+  * Dropped the dependency on libflatpak, as it no longer seems to be used.
```

Comment:
The jenkins build fails for the same exact reason.
```
panels/user-accounts/meson.build:166:0: ERROR: Dependency "flatpak" not found, tried pkgconfig
```

We still need flatpak for our parental controls feature.
Please refer to "Add App Permissions section" commit in the code branch.

Reply:
You are correct, I reinstated this patch. I was misled by the commit message saying it was needed for the companion app service, and by grepping for `flatpak` instead of `Flatpak` :smile: